test/e2e/lib/pwBase.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-empty-pattern */
```

Comment:
Regarding this `pwBase.ts` filename, we use lowercase filenames throughout Calypso.

Reply:
rhanks, updated